base_classes/NXobject.nxdl.xml

```diff
@@ -30,5 +30,25 @@
 		This is the base object of NeXus
 	</doc>
         <!--attribute name="name"><doc>name of instance</doc></attribute-->
+	<field name="identifierNAME" type="NX_CHAR" nameType="partial">
```

Comment:
This looks like it fits in the list of [reserved prefixes](https://manual.nexusformat.org/datarules.html#reserved-prefixes).  Add it to that list.

Reply:
The other prefixes are only defined in the documentation.  Should we make them available to XML by adding them here, similar to `"identifierNAME"`?  Or do this in a separate issue/branch/PR?

Reply:
I added `identifier` to the reserved prefixes.

Reply:
> The other prefixes are only defined in the documentation. Should we make them available to XML by adding them here, similar to `"identifierNAME"`? Or do this in a separate issue/branch/PR?

I think it is good to add them to `NXobject` directly. That still doesn't automatically make them reserved (it is still technically possible to overwrite them somewhere else), but at least the docs can be clearly defined and then inherited. 

I suggest to do it in a differen PR since for the identifier change we already have a NIAC vote (not sure if the other changes would need a separate one).